README.md

```diff
@@ -1358,6 +1358,9 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [CodeMyUI](https://codemyui.com) - Handpicked collection of Web Design & UI Inspiration with Code Snippets.
   * [ColorKit](https://colorkit.co/) - Create color palettes online or get inspiration from top palettes.
   * [coolors](https://coolors.co/) - Color palette generator. Free.
+  * [Colors & Fonts](https://colorsandfonts.com/) -- Free. Color and typography resources and inspiration ready to copy paste
```

Comment:
11 links just to justify a banner promoting a commercial themes site, this just isnt useful.

Reply:
> 11 links just to justify a banner promoting a commercial themes site, this just isnt useful.

All the resources are 100% Free. I have put my sweat an tears for others to use it for free in exchange of nothing. The banners on my projects are there because I don't have any adverts or tracking stuff, is the least I should be able to have on my own free projects that I have built.

If that's a constrain, I would recommend you to go through all the links you have and you will find worst cases.

Reply:
You are welcome to add a PR that remove other sites with minimal value only to promote a commercial site.  I may have missed some, but wont add these.

Reply:
The same

> You are welcome to add a PR that remove other sites with minimal value only to promote a commercial site. I may have missed some, but wont add these.

I am pretty sure that you can do that yourself. The same as you are not accepting my 100% free and without any kind of constrains for the user to use.